keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/PublishedArtifact.kt

```diff
@@ -52,7 +52,14 @@ data class PublishedArtifact(
   val status: ArtifactStatus? = metadata["releaseStatus"]?.toString()
     ?.let { ArtifactStatus.valueOf(it) }
 
-  val createdAt = metadata["createdAt"]
+  val createdAt = let {
+    if (metadata.containsKey("createdAt") && metadata["createdAt"] != null)
+      metadata["createdAt"]
+    else {
+      //docker artifact createdAt time is under date field
+      metadata["date"]
+    }
+  }
```

Comment:
This could be simplified as:
```suggestion
    metadata["createdAt"]
      //docker artifact createdAt time is under date field
      ?: metadata["date"]
```